### PR TITLE
fix: add Text component to the default theme

### LINKED ISSUE #2440: The `Text` component is themed but not present in the default theme.
### Which package has an issue?

@yamada-ui/react

### Package Version

1.4.8

### Description

The `Text` component is themed but not present in the default theme.

You should create and export `text.ts` in the `theme/components` folder.

### Link to Reproduction

_No response_

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux
- [ ] iPadOS
- [ ] iOS
- [ ] Android

### Do you know how to fix the issue?

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by manasan-iTL**
Closes #2440 

## Description

I add Text component to the default theme ( theme/src/components )

## Current behavior (updates)

The Text component is **not** present in the default theme.

## New behavior

The Text component is present in the default theme.

## Is this a breaking change (Yes/No):

No

## Additional Information
None

### CODE REVIEW

**Review by hirotomoyamada — COMMENTED** [review]
Please run `pnpm changeset`, and select `@yamada-ui/theme` and `minor`.

File: packages/theme/src/components/text.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import type { ComponentStyle } from "@yamada-ui/core"
+
+export const Text: ComponentStyle<"Text"> = {
```
Comment (hirotomoyamada) [inline packages/theme/src/components/text.ts:3]: Please change to

```
export const Text: ComponentStyle<"Text"> = {}
```

**Review by hirotomoyamada — COMMENTED** [review]
Please run `pnpm gen:tokens`, and run `pnpm changeset`, and select `@yamada-ui/core` and `patch`.

**Comment by manasan-iTL** [comment]
@hirotomoyamada 
Thank you for your review!

I have fixed the points you raised.
Could you please review it again?

**Review by haru-036 — APPROVED** [review]

**Review by hirotomoyamada — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**